src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -370,6 +370,10 @@
                 <source>This value is not a valid hostname.</source>
                 <target>Cette valeur n'est pas un nom d'hôte valide.</target>
             </trans-unit>
+            <trans-unit id="96">
+                <source>This collection should contain a number of elements that is a multiple of {{ compared_value }}.</source>
+                <target>Cette collection doit contenir un nombre d'éléments étant un multiple de {{ compared_value }}.</target>
```

Comment:
`Cette collection doit contenir un nombre d'éléments multiple de {{ compared_value }}.`

Reply:
I changed the base English sentence to `The number of elements in this collection should be a multiple of {{ compared_value }}.` thanks to @ciaranmcnulty advice.

Reply:
Can you recheck the French translation please?